docs/custard/scenario-data-seeder.md

```diff
@@ -0,0 +1,46 @@
+Scenario Data Seeding Example usage:
+====================================
+
+> This is useful for seeding data to a colleague without needing to clear their entire database 
+
+* In your Application class, you will need to add the following line to `getCustardCommands()`
+```php
+SeedDemoDataCommand::registerDemoDataSeeder(new MyDescribedSeeder());
```

Comment:
Small disconnect between this and your class below. MyDescribedSeeder vs MyDescribedDataSeeder. 

Not a bigy.